config/config.example.yml

```diff
@@ -502,7 +502,22 @@ notifyMetrics:
     config: 'NOTIFY.outgoing.delivered'
     description: 'admin-notify-dashboard.NOTIFY.outgoing.delivered.description'
 
-
+# Creative Commons metadata fields
+ccLicense:
+  # Icon variant:
+  # 'full' variant shows image, a disclaimer (optional) and name (always), better for the item page content.
+  # 'small' (default) variant shows image and name (optional), better for the item page sidebar
+  variant: small
+  # Field name containing the CC license URI, as configured in the back-end, in the 'dspace.cfg' file, property 'cc.license.uri'
+  # Defaults to 'dc.rights.uri'
+  uriField: dc.rights.uri
```

Comment:
Because this setting should always match the `cc.license.uri` setting in `dspace.cfg`, it'd be **better** to just have the backend share that setting with the user interface.  This is what we do in other areas of the application as well.

For example, the backend already does this with `cc.license.jurisdiction` setting in `dspace.cfg`.  In order to tell the backend to make that setting available, you just need to add it to the list of `rest.properties.exposed` in `rest.cfg` like this: https://github.com/DSpace/DSpace/blob/main/dspace/config/modules/rest.cfg#L54

Then, the UI code is able to **request** that setting from the REST API, like this:
https://github.com/DSpace/dspace-angular/blob/main/src/app/submission/sections/cc-license/submission-section-cc-licenses.component.ts#L310

So, this PR could be simplified if you removed the `uriField` setting here, and instead made a call to:
```
 this.configService.findByPropertyName('cc.license.uri')
```

In general, this PR does seem like a step in the right direction.  So, if you feel you cannot update it to read these values from the REST API, we still can move this forward.  But, I wanted to point out how easy it is to just read these settings from the REST API instead of duplicating them to the UI configs.